src/main/java/slimeknights/tconstruct/smeltery/data/SmelteryRecipeProvider.java

```diff
@@ -313,6 +313,15 @@ private void addSmelteryRecipes(Consumer<FinishedRecipe> consumer) {
                        .pattern("BBB")
                        .unlockedBy("has_item", has(TinkerSmeltery.searedBrick))
                        .save(consumer, location(folder + "lantern"));
+    ShapedRecipeBuilder.shaped(RecipeCategory.REDSTONE, TinkerSmeltery.castingTank.get())
+                       .define('#', TinkerSmeltery.searedBrick)
+                       .define('B', Tags.Items.GLASS)
+                       .define('C', Tags.Items.INGOTS_COPPER)
+                       .pattern("#C#")
+                       .pattern("BBB")
+                       .pattern("#B#")
```

Comment:
Lets make the recipe mirror the drain recipe a bit more, cross the drain and tank recipe. So

```
BGB
CGC
BGB
```
where `B` is bricks, `G` is glass, and `C` is copper ingots.